releases/23.10.3/23103 Release Notes.md

```diff
@@ -0,0 +1,73 @@
+  * Update Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp
+  * Detect whether the Property Editor is being instanced in the Editor or in a Unit Test and handle deletion differently
+  * Fix int conversion warnings.
+    
+* https://github.com/o3de/o3de/pull/17670 - Ensure Collider widgets correctly signal when they're being edited #17670
+* https://github.com/o3de/o3de/pull/17712 - Fix the proportions of the Editor splashscreen for 23.10.3
+* https://github.com/o3de/o3de/pull/17828 - Cherry pick PR #17819 Fix decal material loading when asset is already loaded
+* https://github.com/o3de/o3de/pull/17817 - Cherry-pick PR #17568
+  * Fixes the bug causing asset browser to be unable to delete/move/etc. (#17568)
+  * Fixes Asset Browser cannot delete asset #15671
+  * Adds missing functionality from the Linux ProcessWatcher to indicate the difference between a different problem launching program, and the program not being found.
+  * Fixes the workflow of the Move/Delete/Rename operations in the Asset Browser to properly return the appropriate error when Source Control is not properly configured.
+  * Adds a toggle for source control notification in AP, so that if you toggle the Editor to not use Source Control, the AP will not either.
+  * Uses the same setting in AP as the Editor, to determine whether Source Control should be used or not, and sets it to false by default, just like the Editor.
+
+* https://github.com/o3de/o3de/pull/17816 - Cherry-pick pull request #17424
+  * Fix issue #14877 - mouse wheel being intepreted as x-axis movement on linux (#17424)
+  * Fix issue Linux: mouse_delta_x listens to the mouse wheel input #14877 - mouse wheel being intepreted as x-axis movement on Linux
+  * Fixed existing tests and added a new test case
+ 
+* https://github.com/o3de/o3de/pull/17815 - cherry-pick from dev to point-release, PR #17697 - A few small improvements for the editor startup time
+* https://github.com/o3de/o3de/pull/17814 - Fixes Input handler is handling mouse_delta inputs inconsistently
+* https://github.com/o3de/o3de/pull/17814 - cherry-pick from development to Point Release of PR #17672- Fixes input handler is handling mouse_delta inputs inconsistently
+* https://github.com/o3de/o3de/pull/17813 - cherry-pick crashfix part of PR 17664
+* https://github.com/o3de/o3de/pull/17812 - cherry-pick from development to point-release/23103 PR #17594 Add missing file from installer
+* https://github.com/o3de/o3de/pull/17811 - CHERRY-PICK from dev to point-release/23103 PR #17526 - Startup time improvements on Linux
+* https://github.com/o3de/o3de/pull/17809 - Cherrypick PR 17464 from development to point release #17809 - Fixes the ALT key being held down after alt-tabbing
+* https://github.com/o3de/o3de/pull/17808 - Cherry-pick PR17420 from dev to point-release #17808 - This fixes a potential memory stomp issue, Potential fix for ScriptCanvasDeveloper crash
```

Comment:
I'd replace this section with
```
* https://github.com/o3de/o3de/pull/17817 - Fixes the bug causing asset browser to be unable to delete/move/rename/etc.
  * Adds a toggle for source control notification in AP, so that if you toggle the Editor to not use Source Control, the AP will not either.
  * Uses the same setting in AP as the Editor, to determine whether Source Control should be used or not, and sets it to false by default, just like the Editor.
* https://github.com/o3de/o3de/pull/17816 - Fix mouse wheel being intepreted as x-axis movement on linux
* https://github.com/o3de/o3de/pull/17815 - A few small improvements for the editor startup time
* https://github.com/o3de/o3de/pull/17814 - Fixes Input handler is handling mouse_delta inputs inconsistently
* https://github.com/o3de/o3de/pull/17813 - Fix a potential crash and memory leak
* https://github.com/o3de/o3de/pull/17812 - Add missing cmake preset file from installer
* https://github.com/o3de/o3de/pull/17811 - Asset Processor startup time improvements on Linux
* https://github.com/o3de/o3de/pull/17809 - Fixes the ALT key being held down after alt-tabbing back into the editor
* https://github.com/o3de/o3de/pull/17808 - Fix some random crashes if ScriptCanvasDeveloper Gem is active.
```
for simplicity

Reply:
I agree that these suggestions simplify the notes and still maintain an appropriate level of detail.